adapters/risc0/README.md

```diff
@@ -1,10 +1,10 @@
 # Risc0 Adapter
 
-This package adapts Risc0 version 0.14 to work as a ZKVM for the Sovereign SDK.
+This package adapts Risc0 version 0.18 to work as a ZKVM for the Sovereign SDK.
 
 ## Limitations
 
-Since recursion is not included in the 0.14 release, this adapter is currently limited - individual "slots" may
+Since in-VM recursion is not included in the 0.18 release, this adapter is currently limited. Individual "slots" may
```

Comment:
Risc0 calls this user recursion I believe. We can be consistent if you think it is important. FYI.